DOC-4274: Document new property to disable activity link previews (#78)
exoplatform/exo-documentation: docs/Configuration.rst

       Instructions on how to enable/disable an activity type from
       posting on streams.

    -  :ref:`Enabling/Disabling Activity link preview <Configuration.ActivityLinkPreview>`

>        Instructions on how to enable/disable link preview in activity.

Reviewer: I would shorten it to "How to enable/disable ...", like below.

			   a content is shared in a space.  
			-  **exosocial\\:relationship**: Activity post when two user are
			   connected together.
		
.. _Configuration.ActivityLinkPreview:		
		
			    
==========================================
Enabling/Disabling activity link preview
==========================================

eXo Platform allows you to enable or disable the link preview in an activity 
by defining the parameter ``exo.activity.link.preview.enabled`` 
in :ref:`exo.properties <Configuration.ConfigurationOverview>` file.

.. code::

     #Enable/Disable Activity link preview
     exo.activity.link.preview.enabled=true
     
By default, the parameter is set to true i.e. the activities link preview

> is enabled.

Reviewer: Shouldn't we add screenshots to make it clear what preview is, and what is the result when it is disabled ?

Author: ok, i will apply the patch to make screenshots

			   a content is shared in a space.  
			-  **exosocial\\:relationship**: Activity post when two user are
			   connected together.
		
.. _Configuration.ActivityLinkPreview:		
		
			    
==========================================
Enabling/Disabling activity link preview
==========================================

> eXo Platform allows you to enable or disable the link preview in an activity 

Reviewer: We should add an little introduction to describe what is link preview IMO (maybe there is already a documentation page for that ?), something like : "when posting an activity with a link attached, the activity display a preview for this link by fetching the title, an excerpt and a thumbnail of the linked resource", followed by a screenshot showing an example (with the same link than in the other screenshot to show the difference). WDYT ?

Author: Yep I like the idea, I will update this part

Author: Updated

by defining the parameter ``exo.activity.link.preview.enabled`` 
in :ref:`exo.properties <Configuration.ConfigurationOverview>` file.

.. code::

>      #Enable/Disable Activity link preview

Reviewer: We can remove this comment.

Author: ok

     exo.activity.link.preview.enabled=true
     
By default, the parameter is set to true i.e. the activities link preview
is enabled.

If you disable activities link preview i.e. set the variable 

> ``exo.activity.link.preview.enabled`` to false, links will appears like that on the platform:

Reviewer: ```suggestion
``exo.activity.link.preview.enabled`` to false, activities with links are rendered with the link URL only:
```
